docs/intro/install.rst

```diff
@@ -1,245 +1,56 @@
 
-To get started with virtual environments, see `virtualenv installation instructions`_.
-To install it globally (having it globally installed actually helps here),
-it should be a matter of running::
-
-    $ [sudo] pip install virtualenv
-
-Check this `user guide`_ on how to create your virtualenv.
-
-.. note::
-    If you use Linux or OS X, `virtualenvwrapper`_ is a handy tool to create virtualenvs.
-
-Once you have created a virtualenv, you can install Scrapy inside it with ``pip``,
-just like any other Python package.
-(See :ref:`platform-specific guides <intro-install-platform-notes>`
-below for non-Python dependencies that you may need to install beforehand).
-
-Python virtualenvs can be created to use Python 2 by default, or Python 3 by default. As Scrapy
-only supports Python 3, make sure you created a Python 3 virtualenv.
-
-.. _virtualenv: https://virtualenv.pypa.io
-.. _virtualenv installation instructions: https://virtualenv.pypa.io/en/stable/installation/
-.. _virtualenvwrapper: https://virtualenvwrapper.readthedocs.io/en/latest/install.html
-.. _user guide: https://virtualenv.pypa.io/en/stable/userguide/
-
-
-.. _intro-install-platform-notes:
-
-Platform specific installation notes
-====================================
 
+.. _install-conda:
 .. _intro-install-windows:
 
-Windows
--------
+Installing from conda-forge
+===========================
 
-Though it's possible to install Scrapy on Windows using pip, we recommend you
-to install `Anaconda`_ or `Miniconda`_ and use the package from the
-`conda-forge`_ channel, which will avoid most installation issues.
-
-Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
+If you are using Anaconda_ or Miniconda_, you can install Scrapy from the
+`conda-forge`_ channel::
 
   conda install -c conda-forge scrapy
 
-
-.. _intro-install-ubuntu:
-
-Ubuntu 14.04 or above
----------------------
-
-Scrapy is currently tested with recent-enough versions of lxml,
-twisted and pyOpenSSL, and is compatible with recent Ubuntu distributions.
-But it should support older versions of Ubuntu too, like Ubuntu 14.04,
-albeit with potential issues with TLS connections.
-
-**Don't** use the ``python-scrapy`` package provided by Ubuntu, they are
-typically too old and slow to catch up with latest Scrapy.
-
-
-To install Scrapy on Ubuntu (or Ubuntu-based) systems, you need to install
-these dependencies::
-
-    sudo apt-get install python3 python3-dev python3-pip libxml2-dev libxslt1-dev zlib1g-dev libffi-dev libssl-dev
-
-- ``python3-dev``, ``zlib1g-dev``, ``libxml2-dev`` and ``libxslt1-dev``
-  are required for ``lxml``
-- ``libssl-dev`` and ``libffi-dev`` are required for ``cryptography``
-
-Inside a :ref:`virtualenv <intro-using-virtualenv>`,
-you can install Scrapy with ``pip`` after that::
-
-    pip install scrapy
-
-.. note::
-    The same non-Python dependencies can be used to install Scrapy in Debian
-    Jessie (8.0) and above.
-
-
-.. _intro-install-macos:
-
-Mac OS X
---------
```

Comment:
The problem with removing this section is that it is not covered by any links here. If I recall correctly, they were added to remedy a constant stream of tickets related to installation - for many people making Scrapy project is the first time they're using Python.

+1 to simplify it, maybe mention `xcode-select --install` and ask to use virtualenv.